### PR TITLE
Add example .prettierignore file

### PR DESCRIPTION
**Pull request by Fallenstedt**
Adds more detail on how to use .prettierignore so people like me don't clog up issues with simple questions.

#3970

### CODE REVIEW

File: docs/ignore.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ Prettier offers an escape hatch to ignore a block of code or prevent entire file
 
 To exclude files from formatting, add entries to a `.prettierignore` file in the project root or set the [`--ignore-path` CLI option](cli.md#ignore-path).
 
+For example: 
+
+```sh
+# .prettierignore
+# ignores files with an .html extension
+*.html
```
Comment (suchipi) [inline docs/ignore.md]: Prettier doesn't format HTML out of the box, so this is a bit misleading- could we maybe change the example to `*.test.js`?

Comment (suchipi) [reply docs/ignore.md]: (Not that choosing not to format your tests is very common. Not sure what a better glob would be...)

Comment (j-f1) [reply docs/ignore.md]: `*.min.js`?

Comment (azz) [reply docs/ignore.md]: And/or
```js
dist
package.json
```

**Comment by azz** [comment]
Ping @Fallenstedt. Would you be able to update the PR or check the "Allow edits from maintainers" checkbox?

**Comment by Fallenstedt** [comment]
@azz This is from an `unknown repository` and I cannot find the "Allow edits from maintainers checkbox". I have no way of updating this PR without creating a new PR. Is that ok?

**Comment by j-f1** [comment]
Yeah, let’s close this and open a new one, since I don’t think there’s a way to change this PR now.

**Pull request closed without merging**